core/types/src/lib.rs

```diff
@@ -3,14 +3,17 @@
 //! Rust wrappers for SGX types
 
 #![no_std]
+#![feature(core_intrinsics)]
```

Comment:
I assume using `bitflags` would eliminate the need for this (separately, if you are using a feature, document why you're using it, so we know when we can remove it---every feature means we're stuck on nightly until it can be removed).

Reply:
added a comment.
This is needed for `::core::intrinsics::discriminant_value()`